testing/test_pytest_html.py

```diff
@@ -12,6 +13,20 @@
 
 pytest_plugins = ("pytester",)
 
+if os.name == "nt":
```

Comment:
If I remember way that is not the ideal way to test for windows as it produces different results based on which kind of python you run with. My impression is that platform.system() is more reliable: https://github.com/pycontribs/subprocess-tee/blob/main/lib/subprocess_tee/__init__.py#L22